scripts/uberspace-setup-app

```diff
@@ -15,8 +15,8 @@ printf "\n✅  Done!\n"
 printf "\n\n🔷  Setup service...\n"
 cat <<EOT >> ~/etc/services.d/100eyes.ini
 [program:100eyes]
-directory=$(ENV_HOME)s/100eyes
-command=bin/rails server -b 0.0.0.0
+directory=$HOME/100eyes
```

Comment:
`$(ENV_VAR)/s` actually is the syntax to use with supervisord.

http://supervisord.org/configuration.html#environment-variables

Reply:
I thought you would run the script via bash and then at least for me, the `ENV_HOME` would get substituted with empty string.